source/Willow/ReflectiveCascadingStyleSheetBuilder.extension.st

```diff
@@ -0,0 +1,7 @@
+Extension { #name : #ReflectiveCascadingStyleSheetBuilder }
+
+{ #category : #'*Willow' }
+ReflectiveCascadingStyleSheetBuilder >> classification [
```

Comment:
Here we have classification, which is the equivalent of `Classification >> #willow`?

Reply:
No, it's Classification.